monoloco/run.py

```diff
@@ -47,14 +47,20 @@ def cli():
     show.cli(parser)
     visualizer.cli(parser)
 
+    # Monoloco
+    predict_parser.add_argument('--activities', nargs='+',
+                                help='Choose activities to show: social_distance, raise_hand')
     predict_parser.add_argument('--mode', help='keypoints, mono, stereo', default='mono')
     predict_parser.add_argument('--model', help='path of MonoLoco/MonStereo model to load')
     predict_parser.add_argument('--net', help='only to select older MonoLoco model, otherwise use --mode')
     predict_parser.add_argument('--path_gt', help='path of json file with gt 3d localization')
+                                #default='data/arrays/names-kitti-200615-1022.json')
     predict_parser.add_argument('--z_max', type=int, help='maximum meters distance for predictions', default=100)
     predict_parser.add_argument('--n_dropout', type=int, help='Epistemic uncertainty evaluation', default=0)
     predict_parser.add_argument('--dropout', type=float, help='dropout parameter', default=0.2)
     predict_parser.add_argument('--show_all', help='only predict ground-truth matches or all', action='store_true')
+    predict_parser.add_argument('--webcam', help='monstereo streaming', action='store_true')
```

Comment:
Well `--scale` is more for the resolution, the problem is that to convert it to `--long_edge` I would need the image size, I think it would be a bit messy, I'm not sure it is worth the hassle, what do you think ?

Reply:
Ok fine for me, but does it already work for normal predictions as well?